package.json

```diff
@@ -7,7 +7,7 @@
     "start": "webpack serve --open",
     "test": "npm run test:unit; npm run test:integration; npm run test:e2e",
     "test:unit": "jest",
```

Comment:
this `test:unit` should be changed to not include the ui integration tests.

something like: `jest --testPathIgnorePatterns=bar`